# Configures the transmitted radar power (Tx
# attenuation). The output RCS of cluster and objects
# will be compensated for this attenuation. Reducing
# the output power can improve detection in case of
# close range scenarios or inside rooms.
#减少雷达的功率能够提高检测近物和室内场景的效果
# Standard
# -3dB Tx gain
# -6dB Tx gain
# -9dB Tx gain

uint8 radar_power
---